src/strategies/ESPNOW/ESPNOW.h

```diff
@@ -36,7 +36,7 @@
   #define EN_RECEIVE_TIME 0
 #endif
 
-#define EN_MAGIC_HEADER (uint8_t*)"\xEE\xFE\x0E\xEF"
```

Comment:
The magic header is there to identify PJON traffic, not sure if it has sense to be removed.